frontend/src/components/pages/ImportStoryPage.tsx

```diff
@@ -0,0 +1,184 @@
+              <Input id="youtube-link" size="md" type="text" />
+            </FormControl>
+            <FormControl marginTop="40px">
+              <FormLabel
+                fontWeight="bold"
+                htmlFor="languages"
+                marginBottom="15px"
+              >
+                Languages to Exclude
+              </FormLabel>
+              <Box width="300px">
+                <Select
+                  components={{ DropdownIndicator }}
+                  id="languages"
+                  placeholder="Select one or more languages"
+                  styles={colourStyles}
+                />
+              </Box>
+            </FormControl>
+          </Flex>
+          <Flex direction="column" paddingLeft="40px" width="50%">
+            <Text fontWeight="bold" marginBottom="15px">
+              Instructions
+            </Text>
+            <Text fontSize="sm">
+              Please divide the story content into digestible sections for the
+              translator. To create a new section, skip a line between the
+              previous section and the new section by pressing “Enter” on the
+              keyboard.
+            </Text>
+            <Text fontSize="sm" margin="20px 0 15px 0">
+              Example:
+            </Text>
+            <Box backgroundColor="gray.100" padding="10px 20px">
+              <Text fontSize="sm" marginBottom="10px">
+                PlanetRead is a not-for-profit with the vision of a planet where
+                everyone can read and have access to interesting and affordable
+                reading opportunities in native and any other language(s) of
+                interest.
+              </Text>
+              <Text fontSize="sm" marginBottom="10px">
+                (skipped line)
+              </Text>
+              <Text fontSize="sm">
+                This vision is built around its core innovation, Same Language
+                Subtitling (SLS), and a mission to implement SLS on audio-visual
+                content already watched with a passion by millions of people, in
+                every country, in locally, regionally, nationally, and
+                internationally important languages.
+              </Text>
+            </Box>
+            <FormControl isRequired marginTop="40px">
+              <FormLabel
+                fontWeight="bold"
+                htmlFor="upload-file"
+                marginBottom="15px"
+              >
+                Upload file
+              </FormLabel>
+              <Box
+                backgroundColor="blue.50"
+                border="1px solid"
+                borderColor="blue.100"
+                borderRadius="10px"
+                padding="60px 120px"
+                textAlign="center"
+              >
+                <Icon as={MdFolder} color="blue.100" height={16} width={16} />
+                <Text fontSize="sm" marginBottom="15px">
+                  Drag & drop your files here (.doc, .docx)
```

Comment:
wooops, this isnt updated in the desigs. We can actually only accept 1 file, and only .docx
change the copy here to 
`Drag & drop your story file here (.docx)`